server/code/setup/cpsetup

```diff
@@ -22,12 +22,12 @@ the server functional.
 """
 
 import sys
-import commands
+import subprocess
 import os.path, os
 import socket
 import re
 import time
-import httplib
+import http.client
```

Comment:
this library is Python3 only, but I guess we want this script to work across all Pythons?

```python
try:
    from httplib import HTTPConnection
except ImportError
    from http.client import HTTPConnection
```